src/data/bucket/fill_extrusion_attributes.js

```diff
@@ -6,7 +6,7 @@ export const fillExtrusionAttributes = createLayout([
 ]);
 
 export const centroidAttributes = createLayout([
-    {name: 'a_centroid_pos',  components: 2, type: 'Uint16'}
+    {name: 'aCentroidPos',  components: 2, type: 'Uint16'}
```

Comment:
Unless absolutely necessary, could we defer the rename a bit? Requires some effort in tuning gl-native GL->Metal transpiler. Thanks :)